adapters/ortbbidder/bidderparams/config.go

```
@@ -28,9 +32,9 @@ type BidderConfig struct {
 }
 
 // setRequestParams sets the bidder specific requestParams
-func (bcfg *BidderConfig) setRequestParams(bidderName string, requestParams map[string]BidderParamMapper) {
+func (bcfg *BidderConfig) setRequestParams(bidderName string, requestParams map[string]BidderParamMapper) error {
```

if `bcfg == nil` is this check not need,  let it panic.  BidderConfig should be initialise once at the top.  Remove error return type.  The `setRequestParams` is being called on service initialisation, you won't need this checks on request processing, so let it panic if bidder config is invalid.
Even standard library does not handle this kind of errors.  
Refer [handle method with nil receiver](https://stackoverflow.com/questions/60878082/how-to-handle-method-with-nil-receiver)

---

okays